WooCommerce/Classes/POS/ViewHelpers/CollectCashViewHelper.swift

```diff
@@ -27,18 +29,39 @@ final class CollectCashViewHelper {
             return false
         }
 
-        if inputDecimal.compare(orderDecimal) == .orderedAscending {
+        if inputDecimal < orderDecimal {
             onError(Localization.cashPaymentAmountNotEnough)
             return false
         }
         return true
     }
 
-    private func parseCurrency(_ amountString: String) -> NSDecimalNumber? {
-        currencyFormatter.convertToDecimal(amountString, locale: .current)
+    private func parseCurrency(_ amountString: String) -> Decimal? {
+        // Removes all leading/trailing whitespace, if any
+        let sanitized = amountString.trimmingCharacters(in: .whitespacesAndNewlines)
+
+        // Removes currency symbol
+        let symbol = currencySettings.symbol(from: currencySettings.currencyCode)
+        let stringWithoutSymbol = sanitized.replacingOccurrences(of: symbol, with: "")
+
+        // Configures the formatter as close as possible to use the Store's settings
+        let formatter = NumberFormatter()
```

Comment:
nit;

We could move it to a `var` so it wouldn't be created every time we call `parseCurrency`.

Reply:
Good point!